webapp/advantage/schemas.py

```diff
@@ -70,3 +70,10 @@ class TaxIdSchema(Schema):
     "payment_method_id": String(),
     "country": String(),
 }
+
+invoice_view = {
+    "marketplace": String(
+        validate=validate.OneOf(["", "canonical-ua", "canonical-cube"])
```

Comment:
Should you have canonical-blender here too?

Reply:
The API does not support it yet.